shared/constants/security-provider.ts

```diff
@@ -55,6 +55,7 @@ export enum BlockaidResultType {
   Malicious = 'Malicious',
   Warning = 'Warning',
   Benign = 'Benign',
+  Errored = 'Errored',
```

Comment:
based on the [internal thread](https://consensys.slack.com/archives/C05HZHDS89Z/p1706541505600509?thread_ts=1705393852.556679&cid=C05HZHDS89Z), I believe we will want to use "Error" as the value

Reply:
using `Errored = 'Error'` in https://github.com/MetaMask/metamask-extension/pull/22742